src/staking-v3/components/VoteStakeButtonBg.vue

```diff
@@ -1,16 +1,243 @@
-    style="border: 0; margin: 0; width: 512px; height: 512px"
-  />
+  <div id="iframe" class="wrapper--vote-stake">
+    <div ref="target" class="nt-embed"></div>
+  </div>
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue';
+import { defineComponent, ref, onMounted } from 'vue';
+import * as THREE from 'three';
+
 export default defineComponent({
   setup() {
-    return {};
+    const target = ref();
+
+    const fragmentShader = `
+// Fragment shader
+// Uniforms
+uniform vec2 u_resolution;
+uniform vec2 u_mouse;
+uniform float u_time;
+uniform vec4 u_colors[4];
+uniform float u_blur; 
+uniform bool u_animate;
+uniform float u_animate_speed;
+uniform float u_frequency;
+
+#define S(a,b,t) smoothstep(a,b,t)
+/*
+original_author: Patricio Gonzalez Vivo  
+description: sRGB to linear RGB conversion.
+use: <float|vec3|vec4> srgb2rgb(<float|vec3|vec4> srgb)
+*/
+#ifndef SRGB_EPSILON 
+#define SRGB_EPSILON 0.00000001
+#endif
+#ifndef FNC_SRGB2RGB
+#define FNC_SRGB2RGB
+float srgb2rgb(float channel) {
+    return (channel < 0.04045) ? channel * 0.0773993808 : pow((channel + 0.055) * 0.947867298578199, 2.4);
+}
+vec3 srgb2rgb(vec3 srgb) {
+    return vec3(srgb2rgb(srgb.r + SRGB_EPSILON), 
+                srgb2rgb(srgb.g + SRGB_EPSILON), 
+                srgb2rgb(srgb.b + SRGB_EPSILON));
+}
+vec4 srgb2rgb(vec4 srgb) {
+    return vec4(srgb2rgb(srgb.rgb), srgb.a);
+}
+#endif
+/*
+original_author: Patricio Gonzalez Vivo
```

Comment:
Is it enough to state authors to use this code on the portal? Can you check the license?

Reply:
this `fragmentShader` and `vertexShader` is from original iframe file, so I think it's better to keep it